src/safeds_stubgen/api_analyzer/_api.py

```diff
@@ -297,11 +309,28 @@ class ParameterAssignment(PythonEnum):
     NAMED_VARARG = "NAMED_VARARG"
 
 
+@dataclass(frozen=True)
+class Variance:
+    name: str
+    type: AbstractType
```

Comment:
A "variance" has neither a name nor a type. It looks like you're modeling type parameters instead here.

Reply:
Renamed field to type_parameter and the Variance class to TypeParameter